packages/fluentui/accessibility/src/behaviors/Tree/treeItemBehavior.ts

```diff
@@ -35,7 +35,9 @@ export const treeItemBehavior: Accessibility<TreeItemBehaviorProps> = props => {
           'aria-setsize': props.treeSize,
           'aria-posinset': props.index,
           'aria-level': props.level,
-          ...(props.selectable && { 'aria-selected': props.selected }),
+          ...(props.selectable && {
```

Comment:
can you pls add a specification or description in the comments above?

Reply:
Added `description` comments :) 
Didn't add comments as `specification` because there's unit test targeting this behavior.